### PR TITLE
Use Caddy to create iqe test proxy for ephemeral instances

### PR DESCRIPTION
**Pull request by wottop**
## Description

This proxy will allow us to run iqe tests on a local machine against an Ephemeral instance.
The testing can be done on the command line or in PyCharm and enables line break debugging.

### Setup

1. https://docs.google.com/document/d/1m5ittnVQjPSs2b4HNtCO7QEeykSqsLeXqLJYw8Cmmxw
2. Please comment on the doc for questions or clarifications.

### Steps

1. Documented instructions 

### Verification

1. Once set up, all ephemeral specified tests should run successfully. This mirrors our CI configuration.

### CODE REVIEW

File: bin/iqe-ee-proxy.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+oc create configmap caddyfile \
+    --from-file=${TMP_DIR}/Caddyfile
+cat $(dirname $0)/caddy.yaml | oc apply -f -
+
+oc wait --for=condition=Available deployment/swatch-tally-db
+oc port-forward deployment/swatch-tally-db 5432:5432 &
+
+oc wait --for=condition=Available deployment/caddy
+oc port-forward deployment/caddy 9000:9000 8000:8000 5000:5000
```
[the remark this answers is not in the document]
Comment (wottop) [reply bin/iqe-ee-proxy.sh:61]: It's on line 51

**Review by kflahert — APPROVED** [review]

**Comment by kflahert** [comment]
> this worked locally but I ran into an issue with running the tests in pycharm that I want to update in the doc.
> 
> I had to copy the oc binary into /.iqe-env/bin/ or else the tests would fail with an error saying 'oc is an unknown command' This could just be a me issue but figured we should document it

Update: this was caused by using the flatpak version of pycharm. Installing the version from intellij website fixed it

**Review by kartikshahc — APPROVED** [review]

**Review by ntkathole — APPROVED** [review]

**Pull request merged**